apps/crn-frontend/src/network/Network.tsx

```diff
@@ -43,6 +49,7 @@ const Network: FC<Record<string, never>> = () => {
       .then(loadUserProfile)
       .then(loadGroupProfile)
       // Can only be loaded by navigating to the URL
+      .then(loadWorkingGroupList)
```

Comment:
I wonder if this line shouldn't be more at the top near `loadUserList` and `loadGroupList`. From what I understood from the comments, we want to load the content from tabs first, is this right?